testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1218,3 +1219,30 @@ def test_record_type_sel():
 
     assert len(u.select_atoms('name CA and not record_type HETATM')) == 30
     assert len(u.select_atoms('name CA and record_type HETATM')) == 2
+
+
+def test_element_sel():
+    # test auto-topattr addition of string
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("element H")) == 8
+    assert len(u.select_atoms("same element as index 12")) == 8
+
+
+def test_chain_sel():
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("chainID A")) == len(u.atoms)
+
+
+def test_mass_sel():
+    # test auto-topattr addition of float (FloatRangeSelection)
+    u = mda.Universe(TPR)
+    assert len(u.select_atoms("mass 0.8-1.2")) == 23853
```

Comment:
Maybe use "mass 0.8 : 1.2" and encourage use of the alternative "to" operator for numbers to make it look less confusing.